src/utils/dateUtils.ts

```diff
@@ -0,0 +1,17 @@
+const formatDate = (date: Date): string => {
+  const year = date.getFullYear().toString().substring(2, 4);
+  const month = (date.getMonth() + 1).toString().padStart(2, '0');
+  const day = date.getDate().toString().padStart(2, '0');
+
+  return `${year}년 ${month}월 ${day}일`;
+};
+
+const inboxDate = (date: Date): string => {
+  return `${formatDate(date)}에 받은 편지`;
+};
+
+const sendDate = (date: Date): string => {
+  return formatDate(date);
+};
+
+export { inboxDate, sendDate };
```

Comment:
P4;
Date 객체를 받아서 화면에 보여줘야 할 템플릿 형태를 반환하는 `formatDate` 유틸 함수를 만드신 부분이 재활용하기에 무척 좋다고 생각해요!! 👍👍

그런데, `inboxDate` 나 `sendDate` 라는 따로 유틸 함수가 없어도 컴포넌트 단에서 직접 `formatDate` 를 호출하는 걸로 충분히 쉽게 만들어낼 수 있는 값일 것 같아서 없어도 되지 않을까? 싶어요!!

```suggestion
export { formatDate };
```